src/Kunstmaan/AdminBundle/DependencyInjection/KunstmaanAdminExtension.php

```diff
@@ -47,6 +47,10 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.yml');
+
+        if (!empty($config['enable_console_listener']) && $config['enable_console_listener']) {
```

Comment:
`enable_console_listener` should be replaced by `enable_console_exception_listener`